content/source/docs/enterprise/getting-started/policies.html.md

```diff
@@ -0,0 +1,76 @@
+
+## About Sentinel Policies
+
+_Policies_ in TFE are composed of [Sentinel policies](../sentinel/index.html) with some extra features. [Sentinel](https://www.hashicorp.com/sentinel) is an embedded policy-as-code framework integrated with the HashiCorp Enterprise products. It enables fine-grained, logic-based policy decisions, and can be extended to use information from external sources. Within TFE, you can use Sentinel to apply checks to your runs.
+
+A policy consists of:
+
+- The Sentinel policy code
+- An enforcement mode that changes how a policy affects the run lifecycle
+
+Currently, policies in an organization apply to **all** workspaces. You'll want to consider that when writing your policies so they won't fail unrelated workspace runs or configurations. In the future, TFE will provide a way to manage policies so you can apply them per-workspace.
+
+## Creating a Policy
+
+First, make sure you're viewing the organization settings. If you're still on a run page (or any other page), click the "Settings" button in the top navigation bar.
+
+![navigate](./images/policy-navigate.png)
+
+The policy list shows all of the policies you have access to; if you haven't created any, it's empty.
+
+To create your first policy, click the "Create new policy" button in the upper right.
+
+![new button](./images/policy-new-button.png)
+
+On the "Create a new Policy" page, you need to enter at least two items: a policy name, and the policy code. When you've finished, click the "Create Policy" button.
+
+![fields on new policy page](./images/policy-new-fields.png)
+
+### Policy Name
+
+A policy name should tell your colleagues what the policy is for. Most policies are _a particular environment_ of _a particular Terraform configuration,_ so the name should include both the name of the configuration and the name of the environment.
+
+For this example, we'll just create a sample passthrough policy that will allow all runs to "PASS" our policy check. 
+
+In this example, we're using a configuration named "minimum" and we're deploying it in a production environment, so we named it `passthrough`.
+
+### Enforcement Mode
+
+Enforcement Mode alters how a policy result affects your run; "hard-mandatory" will always stop a run if a policy fails, "soft-mandatory" will pause a run and allow a failure to be overridden, and "advisory" will log failures but always allow a run to continue. Use "soft-mandatory" for now so we can see
+
+### Policy Code
+
+You can paste the following code into the code input box. It always resolves to `true` and will allow all runs to pass. This is a good way to see policy checks applied to your run without having a policy "FAIL".
+
+```python
+main = rule { true }
+```
+
+Later, you can switch `true` to `false` to see how a "FAIL" during a policy check can affect your run. You can find many more examples in our [Example policies doc](../sentinel/examples.html).
```

Comment:
> Example policies doc

Can we change this to "Example Policies section"?

Also, can this be changed to an absolute link within the site? (`/docs/enterprise/sentinel/examples.html`)

Reply:
I thought I saw other pages using similar relative links. Will recheck and fix. Thanks.

Reply:
We seem to only use absolute links like that in the nav, but relative in the actual document pages. Any particular reason to make it absolute? I used relative since I was working in the context of this area of the site and it made sense to me at the time.

Reply:
My (personal) use cases for doing "absolute" relative links:

* It ensures moving the doc in question around does not break the link
* Easy copy/paste when making documentation (visit page you want to link, grab link, paste, edit out protocol/host so just URI remains)
* It has prior art in the style guide

I think we need to come to a generally acceptable consensus on this, but the results would probably be codified in the style guide right now which mainly documents relative links using full URIs.